pip/pep425tags.py

```diff
@@ -158,6 +158,8 @@ def get_supported(versions=None, noarch=False):
 
     abis.append('none')
 
+    arches = []
+
     if not noarch:
```

Comment:
I don't quite follow; I thought that since this variable is a list, having it as plural `arches` would make sense. Could you spell out what you would switch the affected variables' names to?